src/lib/context-menu.view.ts

```diff
@@ -38,7 +38,7 @@ export namespace ContextMenu {
         }: {
             state: State
             zIndex?: number
-            [_key: string]: any
+            [_key: string]: any // eslint-disable-line @typescript-eslint/no-explicit-any  -- This is Hotfix
```

Comment:
Instead of '...rest', I believe you can provide the parameter 'style?:{[k:string]:unknown}' (only 'style' is used from '...rest')